multinode_integration_tests/src/mock_blockchain_client_server.rs

```diff
@@ -275,10 +299,10 @@ impl MockBlockchainClientServer {
     }
 
     fn handle_unparsed(conn_state: &mut ConnectionState) -> Option<String> {
-        match conn_state.request_accumulator.find("\n\n") {
+        match conn_state.request_accumulator.find("\r\n\r\n") {
             None => None,
-            Some(newline_offset) => {
-                let content_offset = newline_offset + 2;
+            Some(crlf_offset) => {
```

Comment:
Feel free to ignore this.

I knew that this must be some sort of IT jargon so I googled crlf and now I see. I'm just saying I couldn't recognize the meaning without that help. That's my fault not yours and of your code.

Reply:
Sorry about that. Yeah, CR/LF is moderately standard terminology, but of course the slash isn't syntactic, so I left it out. It's generally pronounced "CUR-liff."